doc/source/user_guide/skimage2_migration.md

```diff
@@ -8,24 +8,93 @@ This document is a work in progress and still subject to change.
 
 scikit-image is preparing to release version 2.0 as a new package: `skimage2`.
 Alongside skimage2, we will release version 1.0.0. Versions 1.x will be using the current API.
-Versions 1.1.x will throw a FutureWarning upon import, as a means to notify users that
+Versions 1.1.x will throw a `FutureWarning` upon import, as a means to notify users that
 they should either upgrade to skimage2 or pin to version 1.0.x.
 
 We have undertaken this to make some long-outstanding, backward-incomptible changes to the scikit-image API.
 Most changes were difficult or impossible to make using deprecations alone.
-To honor the Hinsen principle—that is, never change results silently, unless to fix a bug—we introduce a new package, which gives users an explicit way of upgrading.
+To honor the Hinsen principle---that is, never change results silently, unless to fix a bug---we introduce a new package, which gives users an explicit way of upgrading.
 Users also have the option to use the two versions side-by-side while they do so.
 
 You can find a more detailed description of our motivation and discussion leading up to this in {doc}`SKIP 4 <../skips/4-transition-to-v2>`.
 
-## Changes in skimage2
+(enable-skimage2-warnings)=
 
-_To be defined._
+## Enable skimage2 related warnings
+
+Even before the release of skimage2, you can enable early warnings to prepare your code.
+Run the following [warnings filter](https://docs.python.org/3/library/warnings.html#the-warnings-filter) before you use scikit-image in your code:
+
+```python
+import warnings
+import skimage as ski
+warnings.filterwarnings(action="default", category=ski.util.PendingSkimage2Change)
+```
+
+This will raise warnings for all code that needs to be updated to work as before with the new skimage2 API.
```

Comment:
```suggestion
This will raise a warning for any piece of code which needs updating to continue working with the new (skimage2) API the exact same way it used to (in the old API).
```